ibis/pandas/execution/generic.py

```diff
@@ -45,6 +45,11 @@ def execute_node_literal_value_datatype(op, value, datatype, **kwargs):
     return value
 
 
+@execute_literal.register(ops.Literal, dt.DataType)
```

Comment:
This is necessary for aggregate literal types like arrays, because their underlying data are not hashable so they cannot be added to the dependency graph and therefore cannot be passed as inputs to an `execute_node` rule.